scheduler/config.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"log"
+
+	"github.com/BurntSushi/toml"
+)
+
+// TODO: Vendor dependencies
+
+type grpcServerConfig struct {
```

Comment:
Yes, we recommend making a separate path on actual auth/authz and keep in mind that we are going to pay extra attention to the way you set up TLS, trust and authz itself.